### ISSUE TITLE
No resolved status

### ISSUE THREAD

**Issue by caleuanhopkins** [jayfk/statuspage#99]
I installed the system yesterday, everything is running fine but I have no label for "resolved" status. Is there anyway I can add this staus in manually?

**Comment by marceldiass** [comment]
@caleuanhopkins When you close a issue, the statuspage generator will show it as resolved.

**Comment by caleuanhopkins** [comment]
@marceldiass tried that but the issue closes in Github and disappears from statuspage after running the update command. The page just displays the "no issues in the last 90 days" message.

https://github.com/caleuanhopkins/binkystatus/issues?q=is%3Aissue+is%3Aclosed
http://status.binkytreats.com/

I have no resolved label in the issues section too so there's no label to attach to the issue when it's closed

**Comment by MightyRevenge** [comment]
This happens to me as well. It disappears from historical.

**Comment by caleuanhopkins** [comment]
@MightyRevenge yeah I now have same thing happen. My old issues now appear as "Resolved" but have to run the update command twice

**Comment by ryanvanderpol** [comment]
This is happening to me, too.  I tried manually creating a "resolved" label first because I couldn't figure out how to get that to show up, but when I did I got the error message found in #96. Then I tried closing the issue and it disappeared from the list and it just now says "no issues in the last 90 days"

**Comment by orels1** [comment]
I opened PR with the fix https://github.com/jayfk/statuspage/pull/149
